Configuration/StandardSequences/python/SimL1EmulatorRepack_CalouGT_cff.py

```diff
@@ -102,7 +102,7 @@
 simCaloStage2Layer1Digis.hcalToken = 'unpackHcal'
 
 # uGT inputs for Muons are from unpacked
-simGtStage2Digis.MuonInputTag   = "unpackGtStage2","Muon"
+simGtStage2Digis.MuonInputTag   = "unpackGtStage2:Muon"
```

Comment:
It appears this one line change has broken two unit tests: testTauEmbeddingProducers and testPhase2PixelNtuple. The error messages are both from unrunnable schedules

```
----- Begin Fatal Exception 23-Feb-2019 00:28:19 CET-----------------------
An exception of category 'ScheduleExecutionFailure' occurred while
   [0] Calling beginJob
Exception Message:
Unrunnable schedule
Module run order problem found: 
simGtExtFakeStage2Digis consumes tcdsDigis, tcdsDigis consumes rawDataCollector, rawDataCollector after castorRawData [path digi2raw_step], castorRawData after rpcpacker [path digi2raw_step], rpcpacker after dtpacker [path digi2raw_step], dtpacker after cscpacker [path digi2raw_step], cscpacker after hcalRawDataVME [path digi2raw_step], hcalRawDataVME after esDigiToRaw [path digi2raw_step], esDigiToRaw after ecalPacker [path digi2raw_step], ecalPacker after SiStripDigiToRaw [path digi2raw_step], SiStripDigiToRaw after siPixelRawData [path digi2raw_step], siPixelRawData after gtStage2Raw [path digi2raw_step], gtStage2Raw consumes simGtExtFakeStage2Digis
 Running in the threaded framework would lead to indeterminate results.
 Please change order of modules in mentioned Path(s) to avoid inconsistent module ordering.
----- End Fatal Exception -------------------------------------------------
23-Feb-2019 00:28:19 CET  Closed file file:lhe_and_cleaned.root
---> test testTauEmbeddingProducers had ERRORS
```

and
```
----- Begin Fatal Exception 23-Feb-2019 00:10:33 CET-----------------------
An exception of category 'ScheduleExecutionFailure' occurred while
   [0] Calling beginJob
Exception Message:
Unrunnable schedule
Module run order problem found: 
rawDataCollector consumes gtStage2Raw, gtStage2Raw consumes simGtExtFakeStage2Digis, simGtExtFakeStage2Digis consumes tcdsDigis, tcdsDigis consumes rawDataCollector
 Running in the threaded framework would lead to indeterminate results.
 Please change order of modules in mentioned Path(s) to avoid inconsistent module ordering.
----- End Fatal Exception -------------------------------------------------
23-Feb-2019 00:10:33 CET  Closed file root://eoscms.cern.ch//eos/cms/store/user/cmsbuild/store/relval/CMSSW_10_0_0_pre1/RelValSingleMuPt10/GEN-SIM/94X_upgrade2023_realistic_v2_2023D21noPU-v2/10000/F2B83850-E6CE-E711-8185-0CC47A78A4B0.root
Failure running cmsRun phase2_digi_reco_pixelntuple_cfg.py 0: status 81
status = 20736

---> test testPhase2PixelNtuple had ERRORS
```

Reply:
@Dr15Jones do you think that the problem is there? My understanding was that the problem is due to another PR #25973, and indeed if I test locally its revert #26003 I manage to get the two failing unit tests to succeed...

Reply:
@fabiocos I would trust your test over my guess.